drivers/pwm/pwm_mcux_sctimer.c

```diff
@@ -75,39 +124,65 @@ static int mcux_sctimer_pwm_set_cycles(const struct device *dev,
 		return 0;
 	}
```

Comment:
Should we return an error if duty_cycle requested is 0 and we are unable to process it?

Reply:
I don't believe that will happen- in the case where one channel is setup, we go into this code block. Otherwise, we simply update the duty cycle to 0 on line 184, here: https://github.com/zephyrproject-rtos/zephyr/pull/66301/files/9e68e3a11a312bd23576902bd41e48fdf687d6a4#diff-dd625dd46514efba61daa8b5a67ffb1d559a3c90c3e9f21fcb918792cd70c4cdR184

So either way, I think that we will be able to handle a duty cycle of 0